atomic_reactor/utils/resource_manager.py

```diff
@@ -0,0 +1,187 @@
+"""
+Copyright (c) 2022 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+import os
+import logging
+import json
+import paramiko
+from datetime import datetime
+from shlex import quote
+
+ROOT_PATH = 'resource_manager/'
+logger = logging.getLogger(__name__)
+
+
+class Host:
+    '''
+    Remote host abstract class
+    '''
+    def __init__(self, username: str, hostname: str, arch: str):
+        self.username = username
+        self.hostname = hostname
+        self.arch = arch
```

Comment:
I still think that arch shouldn't be part of host, we know it in advance, we can just create multiple host pools and save another searching parameter.

Reply:
removed